riscv-executor/src/lib.rs

```diff
@@ -61,6 +68,26 @@ impl<F: FieldElement> Elem<F> {
         }
     }
 
+    pub fn from_u32_as_fe(value: u32) -> Self {
+        Self::Field(F::from(value))
+    }
+
+    pub fn from_i64_as_fe(value: i64) -> Self {
+        Self::Field(F::from(value))
+    }
+
+    pub fn from_u64_as_fe(value: u64) -> Self {
+        Self::Field(F::from(value))
```

Comment:
is this checked or unchecked?

Reply:
this one is actually not used, i'll remove it... and I think the other one can also be removed in favor of the 32 bit ones

Reply:
ok